### PR TITLE
workflow/checks: run istio example with tagged image

### PR DESCRIPTION
**Pull request by srenatus**
🚧 ~I want to see this fail first, I'll fix the tag next.~

✔️ Test fails on main, passes on this PR. Timeout set to 10min which be enough to only ever be triggered by problems, never by accident.

### CODE REVIEW

**Comment by srenatus** [comment]
Oh my that's annoying. It'll fail but it takes loooong.

File: .github/workflows/checks.yaml [hunk 1]
```diff
@@ -61,7 +61,13 @@ jobs:
         run: make deploy-ci
 
   e2e:
-    name: Istio End-to-End Test
+    name: Istio End-to-End Test (${{ matrix.variant }})
+    strategy:
+      matrix:
+        include:
+          - variant: main
+          - variant: tag
```
Comment (ashutosh-narkar) [inline .github/workflows/checks.yaml:69]: Can you add a comment describing what this means ?

**Review by ashutosh-narkar — COMMENTED** [review]
Changes look good to me.

File: .github/workflows/checks.yaml (hunk 1 above)
Comment (srenatus) [reply .github/workflows/checks.yaml:69]: Will do!

**Pull request merged**